test/Libraries/RevitNodesTests/Elements/FamilyInstanceTests.cs

```diff
@@ -124,6 +128,61 @@ public void FacingOrientation()
             var dir = famInst.FacingOrientation;
             dir.IsAlmostEqualTo(Vector.ByCoordinates(0.0, 0.0, 1.0));
         }
-        
+
+        [Test]
+        [TestModel(@".\Empty.rvt")]
+        public void CanPlaceFamilyInstanceByTypeAndCoordinateSystem()
+        {
+            // Arrange
+            var famType = ElementSelector.ByElementId(131610, true) as FamilyType;
+            var coordinateSystem = CoordinateSystem.ByOrigin(0, 0)
+                .Rotate(Autodesk.DesignScript.Geometry.Plane.ByOriginNormal(Point.ByCoordinates(0,0), Vector.ByCoordinates(0,0,1)), 45);
+
+            var expectedFacingOrientation = Vector.ByCoordinates(-0.707, 0.707, 0.000);
+
+            // Act
+            var familyInstance = FamilyInstance.ByCoordinateSystem(famType, coordinateSystem);
+            var facingOrientation = familyInstance.InternalFamilyInstance.FacingOrientation;
+
+            // Assert
+            Assert.AreEqual(expectedFacingOrientation.X, facingOrientation.X, Tolerance);
+            Assert.AreEqual(expectedFacingOrientation.Y, facingOrientation.Y, Tolerance);
+            Assert.AreEqual(expectedFacingOrientation.Z, facingOrientation.Z, Tolerance);
+        }
+
+        [Test]
+        [TestModel(@".\FamilyInstance\familyInstanceTests.rvt")]
+        public void CanGetFamilyInstanceRoom()
+        {
+            // Arrange
+            var famInstance = ElementSelector.ByElementId(316162, true) as FamilyInstance;
+
+            var expectedRoomId = 316151;
```

Comment:
does it throw or return null when an instance is not inside a room ? we should probs check for that too